app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form526DocumentUploadFailureEmail < Job
+      STATSD_SENT_METRIC_KEY = 'api.form_526.document_upload_failure_notification_sent'
+      STATSD_EXHAUSTED_METRIC_KEY = 'api.form_526.document_upload_failure_email_job_exhausted'
+
+      # retry for one day
+      sidekiq_options retry: 14
+
+      sidekiq_retries_exhausted do |msg, _ex|
+        job_id = msg['jid']
+        error_class = msg['error_class']
+        error_message = msg['error_message']
+        timestamp = Time.now.utc
+        form526_submission_id = msg['args'][0]
+
```

Comment:
I opted not to use the clunky `Form526JobStatus` creation/update here as we do in other jobs, we will be monitoring these exhaustion handlers in DataDog and I don't see the need to use the status records. Open to being told otherwise.

Reply:
Having, on a couple of occasions, inspected the `Form526JobStatus` records of a `Form526Submission`,  I can see some potential value in recording background job error data, but perhaps others might feel differently.

Reply:
Oh interesting that's good to know. What's a general example of something you had to inspect the status record for you couldn't have just checked in DataDog? Not super familiar with either but forgot you spent quite a bit of time on this stuff

Reply:
Implemented in https://github.com/department-of-veterans-affairs/vets-api/pull/16359/commits/054a135eba5f998c37fa4700f684eb4399cd538a

Reply:
Top of mind would be looking at the job status history for a specific submission, reviewing retry attempts and associated errors, to better understand each step of a job's execution.